### PR TITLE
DATAUP-549: Add import specification endpoint docs

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -733,6 +730,149 @@ Error Connecting to auth service ...
 }
 ```
 
+## Parse import specifications
+
+This endpoint parses one or more import specification files in the staging area into a data
+structure (close to) ready for insertion into the Narrative bulk import cell.
+
+It can parse `.tsv`, `.csv`, and Excel (`.xls` and `.xlsx`) files. Templates for the currently
+supported data types are available in the `import_specifications/templates` directory of this repo.
```
Comment (bio-boris) [inline README.md:739]: can add a link here

Comment (MrCreosote) [reply README.md]: Good idea, added links in a couple other places as well

File: README.md [hunk 2]
```diff
@@ -733,6 +730,151 @@ Error Connecting to auth service ...
...
+**Content example**
+
+```
+GET bulk_specification/?files=file1.<ext>[,file2.<ext>,...]
+```
+`<ext>` is one of `csv`, `tsv`, `xls`, or `xlsx`.
+
+Reponse:
+```
+{"types": {
```
Comment (briehl) [inline README.md:767]: Kinda picky, but all other JSON block examples in the file look like:
```
{
    "key": {
        "other_key": "value"
    }
}
```
I'd vote for the extra return for consistency.

**Review by briehl — CHANGES_REQUESTED** [review]
Just a couple formatting suggestions. Content looks good.

File: README.md [hunk 3]
```diff
@@ -733,6 +730,151 @@ Error Connecting to auth service ...
...
 
+## Parse import specifications
+
+This endpoint parses one or more import specification files in the staging area into a data
+structure (close to) ready for insertion into the Narrative bulk import cell.
+
+It can parse `.tsv`, `.csv`, and Excel (`.xls` and `.xlsx`) files. Templates for the currently
+supported data types are available in the
+[templates](./import_specifications/templates) directory of this repo.
+See the `README.md` file for instructions on template usage.
```
[the remark this answers is not in the document]
Comment (MrCreosote) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (MrCreosote) [reply README.md]: fixed

**Review by briehl — APPROVED** [review]

**Pull request merged**